package.json

```diff
@@ -182,7 +182,7 @@
     "webpack-node-externals": "^1.7.2"
   },
   "resolutions": {
-    "sharp": "0.22.1",
+    "sharp": "0.27.0",
```

Comment:
By updating `Node.js` version, it's also going to be necessary to update `sharp` version, refer to #492